saleor/settings.py

```diff
@@ -949,6 +949,9 @@ def SENTRY_INIT(dsn: str, sentry_opts: dict):
 )
 # Queue name for "async webhook" events
 WEBHOOK_CELERY_QUEUE_NAME = os.environ.get("WEBHOOK_CELERY_QUEUE_NAME", None)
+WEBHOOK_CELERY_DEFERRED_PAYLOADS_QUEUE_NAME = os.environ.get(
+    "WEBHOOK_CELERY_DEFERRED_PAYLOADS_QUEUE_NAME", None
+)
```

Comment:
I think we can just make it default to the webhook queue as I don't believe there is any reason to make it default to the main queue, no?

```suggestion
WEBHOOK_CELERY_DEFERRED_PAYLOADS_QUEUE_NAME = os.environ.get(
    "WEBHOOK_CELERY_DEFERRED_PAYLOADS_QUEUE_NAME", WEBHOOK_CELERY_QUEUE_NAME
)
```